README.md

```diff
@@ -8,8 +8,7 @@ repo](https://github.com/flutter/flutter). It contains the source code for
 Flutter first-party plugins (i.e., plugins developed by the core Flutter team).
 Check the `packages` directory for all plugins.
 
-Flutter plugins enable access to platform-specific APIs. For more information
-about plugins, and how to use them, see
+Flutter plugins enable access to platform-specific APIs. For more information about plugins, and how to use them, see
```

Comment:
You're now removing line wrapping for no reason.

We appreciate the typo fix, as that is a legitimate issue in the documentation, but having lots of review cycles for unrelated irrelevant changes isn't a good use of time. If you can restrict the PR to actual fixes we can move forward, otherwise it'll be best to close this.

Reply:
That was a space formatting mistake by my IDE. I have reversed it back. I am really sorry for the troubles and appreciate your guidance.